packages/babel-parser/test/fixtures/core/uncategorised/263/output.json

```diff
@@ -45,7 +45,8 @@
           "start":0,"end":15,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":15}},
           "extra": {
             "rawValue": "use\\x20strict",
-            "raw": "'use\\x20strict'"
+            "raw": "'use\\x20strict'",
+            "expressionValue": "use strict"
```

Comment:
Note for reviewers: this test shows the difference between `value` and `expressionValue`.